src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SPLIT_SYMBOL = ",";
+    private static final int INDEX_OF_NAME = 0;
+    private static final int INDEX_OF_VALUE = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write(readFromFile(fromFileName).toString());
```

Comment:
you open connection to file while readign  inside of connection for writing, it's better not to open nested connection. Let's separate this (ideally, create private method writeToFile(input, fileName)